examples/styles/lv_example_style_19.c

```diff
@@ -0,0 +1,59 @@
+#include "../lv_examples.h"
+#if LV_BUILD_EXAMPLES && LV_USE_IMAGE
+
+static lv_style_t style1;
+
+static void event_cb(lv_event_t * e)
+{
+    LV_LOG_USER("Clicked");
+
+    static uint8_t change = 0;
+    lv_obj_t * btn = lv_event_get_target(e);
+    lv_obj_t * obj = lv_obj_get_parent(btn);
+    //Change style properties
```

Comment:
Use `/* */` comments
```suggestion
    /* Change style properties */
```

Reply:
Changed it